assets/themes/hailhydrant/hailhydrant.json

```diff
@@ -0,0 +1,601 @@
+                ]
+              },
+              "then": {
+                "en": "The hydrant has one (1) coupling."
+              }
+            },
+            {
+              "if": {
+                "and": [
+                  "couplings=2"
+                ]
+              },
+              "then": {
+                "en": "The hydrant has two (2) couplings."
+              }
+            },
+            {
+              "if": {
+                "and": [
+                  "couplings=3"
+                ]
+              },
+              "then": {
+                "en": "The hydrant has three (3) coupling."
+              }
+            }
+          ]
+        },
+        {
+          "question": {
+            "en": "Update the lifecycle status of the hydrant."
+          },
+          "render": {
+            "en": "Lifecycle status"
+          },
+          "freeform": {
+            "key": "disused:emergency"
+          },
+          "mappings": [
+            {
+              "if": {
+                "and": [
+                  "emergency=fire_hydrant"
+                ]
+              },
+              "then": {
+                "en": "Hydrant is (fully or partially) working."
+              }
+            },
+            {
+              "if": {
+                "and": [
+                  "disused:emergency=fire_hydrant",
+                  "emergency="
+                ]
+              },
+              "then": {
+                "en": "Hydrant is unavailable for use."
+              }
+            },
+            {
+              "if": {
+                "and": [
+                  "removed:emergency=fire_hydrant",
+                  "emergency="
+                ]
+              },
+              "then": {
+                "en": "Hydrant has been removed."
+              }
+            }
+          ]
+        },
+        {
+          "question": {
+            "en": "Describe the position of the hydrant."
+          },
+          "render": {
+            "en": "Position: {fire_hydrant:position}"
+          },
+          "mappings": [
+            {
+              "if": {
+                "and": [
+                  "fire_hydrant:position=sidewalk"
+                ]
+              },
+              "then": {
+                "en": "Sidewalk"
+              }
+            },
+            {
+              "if": {
+                "and": [
+                  "fire_hydrant:position=lane"
+                ]
+              },
+              "then": {
+                "en": "Road-side"
+              }
+            },
+            {
+              "if": {
+                "and": [
+                  "fire_hydrant:position=underground"
+                ]
+              },
+              "then": {
+                "en": "Underground"
+              }
+            }
+          ],
+          "freeform": {
+            "key": "fire_hydrant:position"
+          }
+        },
+        {
+          "render": {
```

Comment:
I'm not really fond of asking a key like this as it isn't particularly user friendly. It is acceptable though, but maybe as last question as not to scare of new contributors?

Even better would be an integration with Mapillary, but that I lack the time for

Reply:
You're right, this is not appropriate for MapComplete contributors, so I'm taking this out